ex16.py

```diff
@@ -0,0 +1,31 @@
+from sys import argv
+
+script, filename = argv
+
+print "We're going to erase %r." % filename
+print "If you don't want that, hit CTRL-C (^C)."
+print "If you do want that, hit RETURN."
+
+raw_input("?")
+
+print "Opening the file..."
+target = open(filename, 'w')
+
+print "Truncating the file. Goodbye!"
+target.truncate()
+
+line1 = raw_input("line 1: ")
+line2 = raw_input("line 2: ")
+line3 = raw_input("line 3: ")
+
+print "Now I'm going to write these to file."
+
+target.write(line1)
+target.write("\n")
+target.write(line2)
+target.write("\n")
+target.write(line3)
+target.write("\n")
+
+print "And finally, we close it."
+target.close()
```

Comment:
Added comment in the file as well.


-- 
Thanks & Regards,
*Pranav Naxane*
Senior Software Engineer
Cuelogic Technologies Pvt. Ltd.
http://www.cuelogic.co <http://www.cuelogic.co.in/>*m*

On 13 January 2016 at 13:03, Onkar Tibe <notifications@github.com> wrote:

> In ex16.py
> <https://github.com/pranavn-cuelogic/python-training/pull/16#discussion_r49558520>
> :
>
> > +
> > +line1 = raw_input("line 1: ")
> > +line2 = raw_input("line 2: ")
> > +line3 = raw_input("line 3: ")
> > +
> > +print "Now I'm going to write these to file."
> > +
> > +target.write(line1)
> > +target.write("\n")
> > +target.write(line2)
> > +target.write("\n")
> > +target.write(line3)
> > +target.write("\n")
> > +
> > +print "And finally, we close it."
> > +target.close()
>
> as we follow process.. can you add what have we learned here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pranavn-cuelogic/python-training/pull/16/files#r49558520>
> .
>